src/components/RequestStatus.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+type RequestStatusProps = {
+  request: {
+    status: RequestStatusOptions
+    message?: string | null
+  }
+  formStatus?: string[] | null
+}
+
+const RequestStatus = ({ request, formStatus }: RequestStatusProps) => {
```

Comment:
Sorry I should have specified. I was referring to the name of the component here, not the prop!
```suggestion
const FormStatus = ({ request, validationErrors }: RequestStatusProps) => {
```